EngineTest/Unit/Segments/SegmentSchemaTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using FlagsmithEngine.Segment.Models;
+using FlagsmithEngine.Segment;
+using FlagsmithEngine.Feature.Models;
+using feature = FlagsmithEngine.Feature;
```

Comment:
I'm not sure what this syntax denotes?

Reply:
it is defined as alias. as we have constant class in both segments and features, so to avoid ambiguity.